schema.prisma

```diff
@@ -522,6 +525,48 @@ model BankTransactionsFile {
   @@map("bank_transactions_files")
 }
 
+model CampaignReport {
+  id                  String               @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  campaignId          String               @map("campaign_id") @db.Uuid
+  campaign            Campaign             @relation(fields: [campaignId], references: [id])
+  startDate           DateTime             @map("start_date")
+  endDate             DateTime             @map("end_date")
+  description         String
+  files               CampaignReportFile[]
+  totalFunds          Int?               @map("total_funds") @default(0)
+  fundsForPeriod      Int?               @map("funds_for_period") @default(0)
+  spentFundsForPeriod Int?               @map("spent_funds_for_period") @default(0)
+  goals               String
+  nextSteps           String               @map("next_steps")
+  additionalInfo      String               @map("additional_info")
+  creatorId           String               @map("creator_id") @db.Uuid
+  creator             Person               @relation(fields: [creatorId], references: [id])
+  isDeleted           Boolean              @map("is_deleted") @default(false)
+
+  @@map("campaign_reports")
+}
+
+enum CampaignReportFileType {
+  photo
+  document
+
+  @@map("campaign_report_file_type")
+}
+
+model CampaignReportFile {
+  id        String                 @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  reportId  String                 @map("report_id") @db.Uuid
+  report    CampaignReport         @relation(fields: [reportId], references: [id])
+  filename  String                 @db.VarChar(200)
+  mimetype  String                 @db.VarChar(50)
+  type      CampaignReportFileType
+  personId  String                 @map("person_id") @db.Uuid
+  person    Person                 @relation(fields: [personId], references: [id])
```

Comment:
[naming convention] let's have these uploadedBy to indicate that they indicate who uploaded them. In other context someone may confuse the file is "about the person".